reco/LST1_Hillas.py

```diff
@@ -205,9 +210,25 @@ def guess_type(filename):
             l = np.rad2deg(np.arctan2(hillas.length, foclen))
 
             #Calculate Timing parameters
-        
-            peak_time = units.Quantity(peakpos[chan])*units.Unit("ns")
-            timepars = time.timing_parameters(geom.pix_x,geom.pix_y,clean,peak_time,hillas.psi)
+#            peak_time = units.Quantity(peakpos[chan])*units.Unit("ns")
+#            timepars = time.timing_parameters(geom.pix_x,geom.pix_y,clean,peak_time,hillas.psi)
+            pedcorrectedsamples[~window] = 0.0            
```

Comment:
lines 215-220 should go in a different library, probably useful for anybody doing analysis on ctapipe